src/Avalonia.Controls/ItemsControl.cs

```diff
@@ -489,18 +489,20 @@ private void UpdateItemCount()
             bool wrap)
         {
             IInputElement result;
+            var c = from;
 
             do
             {
-                result = container.GetControl(direction, from, wrap);
+                result = container.GetControl(direction, c, wrap);
+                from ??= result;
```

Comment:
This is C# 8 feature, I thought we are not using it until next version.

Reply:
This is targeting master so I thought it would be ok to use it, as we've already branched for 0.9. I can change it though.

Reply:
Then shouldn’t https://github.com/AvaloniaUI/Avalonia/pull/303 go in first?  Since your build actually failed on macOS.

Reply:
Reverted that to C#7 syntax.